src/kwlib.sh

```diff
@@ -313,3 +313,17 @@ function store_statistics_data
 
   echo "$label $value" >> "$day_path"
 }
+
+# This function checks if a certain command can be run
+#
+# @command The whole command that is meant to be executed
+function command_exists()
+{
+  local command="$1"
+  local package=( $command )
```

Comment:
Just for curiosity, why do we need `( $command )` why not just `"$command"`?

Reply:
`( $ command )` retrieves the first word of the whole command. So, for instance, suppose we have a command like
`notify-send -i checkbox -t 10000 "kw" "Command: \"$COMMAND\" completed!" &`
If I do `package="$command"`, I'll have the command entirely, and I believe we only need the first word of it. And to get the first word, I did ( $command ). 
Does that make sense?